acrobat/scripts/scripts.js

```diff
@@ -336,3 +336,40 @@ const { ietf } = getLocale(locales);
     }
   }, 1000);
 }());
+
+// Process placeholders in the document
+const processPlaceholders = (documentElement) => {
+  const placeholderRegex = /{{(.*?)}}/g;
+  const matches = [...documentElement.innerHTML.matchAll(placeholderRegex)];
+  const matchingElements = [];
+  if (matches.length > 0) {
+    const paragraphs = documentElement.querySelectorAll('p');
+    paragraphs.forEach((p, index) => {
+      if (p.innerHTML.match(placeholderRegex)) {
+        const translationLabel = p.innerHTML.match(placeholderRegex)[0];
+        const label = translationLabel.slice(2, -2);
+        p.setAttribute('data-local', label);
+        p.setAttribute('id', `para-${index}`);
+        if (label === 'credit-cards') {
+          matchingElements.push(p);
+        }
+      }
+    });
+  }
+  return matchingElements;
+};
+
+// Check if the meta tag named 'commerce' exists and is set to 'true'
+const isCommerceActive = () => {
```

Comment:
this is great @joaquinrivero 
a few minor notes.
1. Let's only keep this block in `scripts.js` and move the rest to into the `imageReplacer.js`